mosec/args.py

```diff
@@ -134,6 +134,13 @@ def build_arguments_parser() -> argparse.ArgumentParser:
         "This will omit the worker number for each stage.",
         action="store_true",
     )
+
+    parser.add_argument(
+        "--compression",
+        help="Enable Zstd & Gzip compression for the request body",
```

Comment:
Zstd & Gzip are enabled together? They are two algorithms as I understand?

Reply:
Yes. The compression layer will choose the algorithm according to the request headers.

I enabled both since gzip is widely used (included in Python std) and zstd is currently the best. Users can choose the one that better suits their use cases.

Reply:
Cool! Can we add an example (similar to the test script) and a pointer in README to make this feature more visible?

Reply:
I'm still unsure what the best use case for compression is in model serving. Numpy vectors to bytes can benefit a little. Images that are using compression (JPEG) should not be applied again. @aseaday do you have suggestions?

Reply:
Numpy vectors can benefit a lot in our cases.
We had a feature image translator in our product [immersivetranslate](https://immersivetranslate.com/).
To fullfill  image translator, there is a ML model pipeline: detection, recgonize and inpaint. It is also a common desgin for image OCR.
The variables between step in this pipeline are a msgpack format numpy data such as a image mask. A image mask is [3,1024,1024] array. Before compression, the file size will be 6MB, after compression, the filee size will be 2MB because of Sparsity